massive-data-import/README.md

```diff
@@ -0,0 +1,191 @@
+# Massive Data Import to Kuzzle
+
+## Requirements
+
+Kuzzle : `>= 1.2.11`  
+
+## Introduction
+
+When processing large volumes of data with Kuzzle, you may need to import large datasets quickly.  
+
+Kuzzle offers two massive data import systems according to your needs:
+- Using [bulk imports](https://docs.kuzzle.io/api-documentation/controller-bulk/import/), when you need to import data as fast as possible
+- With [multi-documents creations](https://docs.kuzzle.io/api-documentation/controller-document/m-create/), a bit slower, but this method allows the use of real-time notifications or plugin events during import
+
+Using an AWS production environment with Kuzzle on a `m5.large`, Elasticsearch on a `i3.xlarge.elasticsearch` and Redis on a `cache.t2.micro` instance, we were able to achieve 9700 docs/sec with Bulk import and 5800 docs/sec with the mCreate route.  
+
+In this How-To, we will explore the two massive import techniques of Kuzzle.
+For this example we will use data from the NYC Yellow Taxi dataset.  
+
+## Architecture
```

Comment:
Information missing: the necessary Kuzzle configuration changes allowing to play the scripts provided in this how-to